src/pheval/analyse/benchmarking_data.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from pheval.analyse.rank_stats import RankStats
+
+
+@dataclass
+class BenchmarkRun:
```

Comment:
As with previous comment  - rename to `BenchmarkRunResults`?

Reply:
Also, can't this file and `benchmark_generator.py` be merged into a single `benchmark.py` module?

Reply:
I can't merge it into a single file as there will be a circular import

Reply:
Where is the circular dependency?

Reply:
The `gene_prioritisation_analysis.py`, `variant_prioritisation_analysis.py` and `disease_prioritisation_analysis.py` import `BenchmarkRunResults` and the benchmark generator imports from each of these files